vagga.yaml

```diff
@@ -1,18 +1,24 @@
 minimum-vagga: v0.7.0
 containers:
-  pip-freeze:
+  pip-freeze-main:
+    setup:
+      - !Alpine v3.4
+      - !Py3Install
+        - pip
+        - 'Django >=1.9,<1.10'
+        - 'django-environ >=0.4,<0.5'
+      - !Sh pip freeze > requirements/main.txt
+  pip-freeze-production:
```

Comment:
Sorry, but this'll not work
`Proceed (y/n)? Exception:
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python3.5/site-packages/pip/commands/uninstall.py", line 76, in run
    requirement_set.uninstall(auto_confirm=options.yes)
  File "/usr/lib/python3.5/site-packages/pip/req/req_set.py", line 346, in uninstall
    req.uninstall(auto_confirm=auto_confirm)
  File "/usr/lib/python3.5/site-packages/pip/req/req_install.py", line 754, in uninstall
    paths_to_remove.remove(auto_confirm)
  File "/usr/lib/python3.5/site-packages/pip/req/req_uninstall.py", line 103, in remove
    response = ask('Proceed (y/n)? ', ('y', 'n'))
  File "/usr/lib/python3.5/site-packages/pip/utils/__init__.py", line 158, in ask
    response = input(message)
EOFError: EOF when reading a line`